ExpressApp/controllers/api-comments-controller.js

```diff
@@ -0,0 +1,20 @@
+const { checkExists } = require("../models/api-checkExists-model");
+const { deleteCommentById } = require("../models/api-comments-model");
+
+exports.deleteComment = (req, res, next) => {
+  const id = req.params.comment_id;
+
+  const commentPromises = [];
+
+  if (id) {
+    commentPromises.push(checkExists("comments", "comment_id", id));
+  }
+
+  commentPromises.push(deleteCommentById(id));
```

Comment:
These can exist in the promise.all to start with. There will always be an ID so we don't need to check if its truthy.